src/common/rbac/project/util.go

```diff
@@ -30,6 +36,20 @@ var (
 	}
 )
 
+func policiesForPublicProjectRobot(namespace rbac.Namespace) []*rbac.Policy {
```

Comment:
`policiesForPublicProjectRobot`,`policiesForPublicProject`,`policiesForSystemAdmin `

The code looks extremely repetitive.